### PR TITLE
doc changes for OKTA-106645

### PR DESCRIPTION
**Pull request by ashishgupta-okta**
doc changes for OKTA-106645

@royalchan-okta @mystiberry-okta 
@federations-okta

### CODE REVIEW

File: _source/_docs/api/resources/oidc.md [hunk 1]
```diff
@@ -71,6 +71,7 @@ to access [the OIDC `userinfo` endpoint](http://developer.okta.com/docs/api/reso
   be a valid OAuth 2.0 request, but it's not an OpenID Connect request.
 * `profile` requests access to these default profile claims: `name`, `family_name`, `given_name`, `middle_name`, `nickname`, `preferred_username`, `profile`,  
 `picture`, `website`, `gender`, `birthdate`, `zoneinfo`,`locale`, and `updated_at`.
+* `offline_access` scope can only be requested in combination with response_type `code`.
```
Comment (ashishgupta-okta) [inline _source/_docs/api/resources/oidc.md]: done

File: _source/_docs/api/resources/oidc.md [hunk 2]
```diff
@@ -71,6 +71,7 @@ to access [the OIDC `userinfo` endpoint](http://developer.okta.com/docs/api/reso
   be a valid OAuth 2.0 request, but it's not an OpenID Connect request.
 * `profile` requests access to these default profile claims: `name`, `family_name`, `given_name`, `middle_name`, `nickname`, `preferred_username`, `profile`,  
 `picture`, `website`, `gender`, `birthdate`, `zoneinfo`,`locale`, and `updated_at`.
+* `offline_access` can only be requested in combination with a response_type containing `code`.
```
Comment (royalchan-okta) [inline _source/_docs/api/resources/oidc.md]: same here, `response_type` needs the quotes, otherwise LGTM

**Comment by mystiberry-okta** [comment]
Looks good, @ashishgupta-okta . Will merge next week for Release .47, let me know if that's not correct.

**Comment by royalchan-okta** [comment]
re 🚢

**Pull request merged**